modules/friendlytag.js

```diff
@@ -959,13 +1035,13 @@ Twinkle.tag.callbacks = {
 						currentTag += '|source=' + params.tagParameters.closeParaphrasing;
```

Comment:
There are a few more of these (seven total?) that still need removing.

```suggestion
						currentTag += '|source=' + params.closeParaphrasing;
```

Reply:
oh, they crept in from the merge commit.